tests/python/test_dynamic.py

```diff
@@ -283,3 +264,80 @@ def make_list():
             assert x[i, j].b == i * j * 10000 % 65536
             assert x[i, j].c == i * j * 100000000 % 4294967296
             assert x[i, j].d == i * j * 10000000000
+
+
+def _test_append_matrix():
+    mat = ti.types.matrix(n=2, m=2, dtype=ti.u8)
+    f = mat.field()
+    pixel = ti.root.dense(ti.i, 10).dynamic(ti.j, 20, 4)
+    pixel.place(f)
+
+    @ti.kernel
+    def make_list():
+        for i in range(10):
+            for j in range(20):
+                f[i].append(
+                    ti.Matrix([[i * j, i * j * 2], [i * j * 3, i * j * 4]],
+                              dt=ti.u8))
+
+    make_list()
+
+    for i in range(10):
+        for j in range(20):
+            for k in range(4):
+                assert f[i, j][k // 2, k % 2] == i * j * (k + 1) % 256
+
+
+@test_utils.test(require=ti.extension.sparse, exclude=[ti.metal])
+def test_append_matrix():
+    _test_append_matrix()
+
+
+@test_utils.test(require=ti.extension.sparse,
+                 exclude=[ti.metal],
+                 real_matrix=True,
```

Comment:
parametrize?

Reply:
Can pytest.parametrize be used on the decorator?

Reply:
@lin-hitonami do you mean https://github.com/taichi-dev/taichi/blob/master/tests/python/test_ndarray.py#L38? ;)

Reply:
I mean we can't use it on parameters in `@test_utils.test()`. (Do you mean we can parametrize `real_matrix=True`?)